CMakeLists.txt

```diff
@@ -8,7 +8,12 @@ endif()
 
 include(SCVersion.txt)
 # CMake versions can only consist of digits and periods, so don't use _TWEAK
-project(SuperCollider VERSION ${SC_VERSION_MAJOR}.${SC_VERSION_MINOR}.${SC_VERSION_PATCH})
+if(STANDALONE)
+	project("SuperCollider-Standalone" VERSION ${SC_VERSION_MAJOR}.${SC_VERSION_MINOR}.${SC_VERSION_PATCH})
```

Comment:
could there be a variable that can be set that will allow us to set the name of the application? This could be "SuperCollider-Standalone" by default, but could be overridden.

Reply:
There was a stub for this (may not have really worked) previously.

Reply:
OK, that is a stumbling block, I suppose.